lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -305,13 +307,38 @@ class StandardAccountTest {
             calldata = calldata,
             maxFee = Felt.ZERO,
         )
-        val fee = provider.getEstimateFee(payloadForFeeEstimation).send()
-        val maxFee = fee.overallFee
+        val feePayload = provider.getEstimateFee(payloadForFeeEstimation).send()
+        assertTrue(feePayload.overallFee.value > Felt.ONE.value)
+        println(feePayload)
+    }
+
+    @Test
+    fun `deploy account`() {
+        val provider = gatewayProvider
+        val privateKey = Felt(11111)
+        val publicKey = StarknetCurve.getPublicKey(privateKey)
+
+        val classHash = accountClassHash
+        val salt = Felt.ONE
+        val calldata = listOf(publicKey)
+        val address = ContractAddressCalculator.calculateAddressFromHash(
+            classHash = classHash,
+            calldata = calldata,
+            salt = salt,
+        )
+        devnetClient.prefundAccount(address)
+
+        val account = StandardAccount(
+            address,
+            privateKey,
+            provider,
+        )
         val payload = account.signDeployAccount(
             classHash = classHash,
             salt = salt,
             calldata = calldata,
-            maxFee = maxFee,
+            // 10*fee from estimate deploy account fee
+            maxFee = Felt.fromHex("0x11fcc58c7f7000"),
```

Comment:
I think it'd be more readable not to use hex here.

Reply:
I can't use integer, because it its bigger than the maximum value. What is not readable?

Reply:
I believe you can create Felt from BigInteger and BigInteger allows construction from decimal string (more readable than hex imo).

Reply:
But it is not important what the value is, it is just a big number. Why do you need this?

Reply:
It's just a little bit easier to read what value is being passed to transaction if it's in decimal.

Reply:
I prefer using hex values, because they are shorter and the actual value doesn't matter.